addons/mail/models/mail_message.py

```diff
@@ -897,7 +897,7 @@ def _record_by_message(self):
                 record_by_message[message] = self.env[message.model].browse(message.res_id).with_prefetch(record_ids_by_model_name[message.model])
         return record_by_message
 
-    def _message_format(self, format_reply=True, msg_vals=None):
+    def _message_format(self, format_reply=True, msg_vals=None, include_internal=False):
```

Comment:
I'd suggest a name like `for_current_user`, `user_specific` or `personalized`, because "internal" doesn't speak to me in this context.

Reply:
I choose `for_current_user` as I think it's explicit enough.